### PR TITLE
Improve debugging messages

### PR DESCRIPTION
**Pull request by markstory**
Add which migration is causing the state to be inconsistent.

Refs #533

### CODE REVIEW

File: src/TestSuite/Migrator.php [hunk 1]
```diff
@@ -160,7 +160,11 @@ protected function shouldDropTables(Migrations $migrations, array $options): boo
 
         foreach ($migrations->status($options) as $migration) {
             if ($migration['status'] === 'up') {
-                Log::write('debug', 'One or more additional migrations detected.');
+                Log::write(
+                    'debug',
+                    'One or more migrations need to be applied. ' .
+                    "Migration with source={$migration['name']} id={$migration['id']} needs to be applied."
```
Comment (othercorey) [inline src/TestSuite/Migrator.php]: Do we have any way to format a list?

Comment (markstory) [reply src/TestSuite/Migrator.php]: Not really. We also break out of the loop early. We could instead collect all the output and format a list.

Comment (othercorey) [reply src/TestSuite/Migrator.php]: Can we at least line break and prefix with something? A `-` or a space or whatever?

**Comment by othercorey** [comment]
@dereuromark can you review this?

File: src/TestSuite/Migrator.php [hunk 2]
```diff
@@ -112,7 +112,7 @@ public function runMany(
 
             if (!$migrations->migrate($migrationSet)) {
                 throw new RuntimeException(
-                    sprintf('Unable to migrate fixtures for `%s`.', $migrationSet['connection'])
+                    "Unable to migrate fixtures for `{$migrationSet['connection']}`."
```
Comment (dereuromark) [inline src/TestSuite/Migrator.php:115]: Personally, I like the sprintf() syntax more, in terms of readability etc. but no blocker.

**Review by dereuromark — APPROVED** [review]

**Pull request merged**